tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -41,11 +40,38 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 runtime = ProcessInfo.Runtime.NetFx;
             }
 
-            var profilerModule = FindProfilerModule(process);
+            bool isContinuousProfilerEnabled;
 
-            if (profilerModule == null)
+            if (process.EnvironmentVariables.TryGetValue("DD_PROFILING_ENABLED", out var profilingEnabled))
```

Comment:
You might want to run all of the ContinuousProfiler checks after identifying the loader module and the native tracer module because these console messages aren't actionable if the profiler isn't a part of the product they installed